src/allmydata/client.py

```diff
@@ -39,9 +52,17 @@
 TiB=1024*GiB
 PiB=1024*TiB
 
-def _valid_config():
-    cfg = node._common_valid_config()
-    return cfg.update(configutil.ValidConfiguration({
+def _is_valid_section(section_name):
+    """
+    Check for valid dynamic configuration section names.
+
+    Currently considers all possible storage server plugin sections valid.
+    """
+    return section_name.startswith("storageserver.plugins.")
+
```

Comment:
This is only called if a static-section *doesn't* match, right? (i.e. it doesn't need to call-through to anything else right?)

Reply:
Correct.  Static checks happen first and the whole thing short-circuits.